gamemode/modules/chat/sv_chat.lua

```diff
@@ -96,15 +96,16 @@ local function RP_ActualDoSay(ply, text, callback)
     callback = callback or ""
     if text == "" then return "" end
     local col = team.GetColor(ply:Team())
-    local col2 = Color(255,255,255,255)
+    local col2 = Color(255, 255, 255, 255)
     if not ply:Alive() then
-        col2 = Color(255,200,200,255)
+        col2 = Color(255, 200, 200, 255)
         col = col2
     end
 
     if GAMEMODE.Config.alltalk then
-        for k, v in pairs(player.GetAll()) do
-            DarkRP.talkToPerson(v, col, callback .. ply:Name(), col2, text, ply)
+        local name = ply:Name()
+        for k, v in ipairs(player.GetAll()) do
+            DarkRP.talkToPerson(v, col, callback .. name, col2, text, ply)
```

Comment:
meh, if you were going to optimise to this level, you could've at least pulled the concatenation out of the loop (or even the if statement). This isn't the only place. I don't think it's worth the effort going back now though.

Reply:
I just corrected what I saw, sorry. :P